src/internal/Scheduling.scala

```diff
@@ -133,7 +133,11 @@ trait Scheduling {
     val symsCache = new IdentityHashMap[Sym[Any], List[Stm]]()
     val boundSymsCache = new IdentityHashMap[Sym[Any], List[Stm]]()
     //val boundSymsCache = new IdentityHashMap[Sym[Any], Set[Stm]]()
-    
+
+    implicit class IdentityHashMapOps[K, V](map: IdentityHashMap[K, V]) {
```

Comment:
Left over from some tests someone was doing comparing different implementations of HashMap on scheduling performance.